cg/cli/workflow/balsamic/base.py

```diff
@@ -213,7 +207,6 @@ def start(
         run,
         case_id=case_id,
         slurm_quality_of_service=slurm_quality_of_service,
-        run_analysis=run_analysis,
```

Comment:
Since we'll start using --dry-run instead of --r, I would remove it from the config_case invocation above to have the same behaviour (we want the config to be created but don't run the analysis).